pyproject.toml

```diff
@@ -35,7 +35,7 @@ requires=
 ; * test - run trial tests
 ; * dev - run things in local dev mode
 envlist =
-    py{35,36,37,38,39,py3}-test-{twlatest,twtrunk,dev}
+    py{36,37,38,39,py3}-test-{twlatest,twtrunk,dev}
```

Comment:
Very minor comment.
I now see it.... maybe have `py-linters` using OS default python.

Recently I started using `tox -p all` as a way to run "default" tests.

Maybe it would be nice to have it in ldapor... but I don't know how to make it work with all the twisted versions.

Maybe in envlinst have `py-test-dev, py-linters,documentation`

Reply:
will mull it over, I think it would need to be a new PR